frontend/dev-mode/src/components/ChatResults.vue

```diff
@@ -0,0 +1,140 @@
+  <div class="flex-1 h-full flex flex-col justify-end" ref="resultsContainer">
+    <template v-for="([actor, response], index) in session.transcript" :key="index">
+      <div class="font-bold mt-6">{{ template(actor) }}</div>
+
+      <MarkdownContent
+        :source="joinResponse(response as TranscriptMessage[])"
+        class="mt-2 bg-elevation-01 px-4 py-2 space-y-[1em]" />
+
+      <CopyTextButton
+        v-if="((response as TranscriptMessage[])[0]).id_slot !== undefined"
+        :text="joinResponse(response as TranscriptMessage[])"
+        class="mt-6">COPY RESULTS</CopyTextButton>
+    </template>
+
+    <LoadingState v-show="isPending" />
+  </div>
+
+  <div class="flex flex-col sticky bottom-0 bg-night pt-8">
+    <form @submit.prevent="send()">
+      <Textarea
+        id="textarea-chat-message"
+        autogrow
+        :persist="false"
+        ref="promptInput"
+        :label="session.user"
+        rows="1"
+        :placeholder="`Message ${session.char}`"
+        v-model="msg"
+        @keydown="onKeyDown">
+        <template v-if="session.image_selected" #before>
+          <div class="relative w-fit">
+            <button tabindex="-1" class="absolute right-0 top-0 h-4 w-4 text-lg bg-gray-05" @click="removeImage">
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor"><path d="M11.9997 10.5865L16.9495 5.63672L18.3637 7.05093L13.4139 12.0007L18.3637 16.9504L16.9495 18.3646L11.9997 13.4149L7.04996 18.3646L5.63574 16.9504L10.5855 12.0007L5.63574 7.05093L7.04996 5.63672L11.9997 10.5865Z"></path></svg>
+            </button>
+            <img :src="session.image_selected" class="max-w-24 max-h-24" alt="image uploaded by the user">
+          </div>
+        </template>
+      </Textarea>
+
+      <div class="flex items-center justify-between">
+        <div class="flex gap-4 mt-6">
+          <Button type="submit" :disabled="!msg.trim() || isGenerating" @click="send()">{{ !isGenerating ? 'SEND' : 'GENERATING...' }}</Button>
+          <Button secondary :disabled="!isGenerating" @click="stop">STOP</Button>
+        </div>
+
+        <label v-if="!isGenerating" class="flex items-center font-bold cursor-pointer gap-2 text-xs hover:text-cornflower">
+          <span>UPLOAD IMAGE</span>
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="25" viewBox="0 0 24 25" fill="none">
+            <path d="M4 19.5H20V12.5H22V20.5C22 21.0523 21.5523 21.5 21 21.5H3C2.44772 21.5 2 21.0523 2 20.5V12.5H4V19.5ZM13 9.5V16.5H11V9.5H6L12 3.5L18 9.5H13Z" fill="currentColor" />
+          </svg>
+          <input type="file" id="fileInput" class="opacity-0 absolute w-px h-px" @change.prevent="uploadImage">
+        </label>
+      </div>
+    </form>
+
+    <div class="mt-4 flex items-center justify-between">
+      <button
+        class="flex items-center gap-2 font-bold hover:text-cornflower"
+        @click="emit('leave')">
+        CHANGE PARAMETER
```

Comment:
This line looks a little weird on the page, should maybe be "Change Parameters"?